docs/src/docs/core/Timeline.mdx

```diff
@@ -34,3 +34,7 @@ Control timeline appearance with the following props:
 ## Bullet as React node
 
 <Demo data={TimelineDemos.bullet} />
+
+## Radius
+
+<Demo data={TimelineDemos.radius} />
```

Comment:
This demo should be a part of configurator

Reply:
Okay I removed it